langworld_db_data/adders/listed_value_adder.py

```diff
@@ -41,14 +60,20 @@ def _add_to_inventory_of_listed_values(
         feature_id: str,
         new_value_en: str,
         new_value_ru: str,
-    ) -> str:
+        index_to_assign: int,
+    ) -> (str, list):
```

Comment:
1. you should include description of what the function returns in its docstring
2. `list` should have typehint for elements of the list
3. it's better to return an immutable sequence (tuple)
4. it's very unclear why the function `_add_to_inventory...` returns some iterable with indexes that should be changed in profiles. Also, the function is pretty long. I recommend that you at least consider renaming it or, if it's possible, splitting it.

Reply:
Everything is done except for function renaming or splitting

Reply:
`_add_to_inventory_of_listed_values` now has two inner methods — one for gathering value IDs and line numbers, one for updating IDs in rows when `index_to_assign != -1` (both are checked with specific tests). Also `_add_to_inventory_of_listed_values` now does not return tuple of IDs to be incremented in feature profiles